include/deal.II/numerics/data_out_dof_data.h

```diff
@@ -854,6 +855,36 @@ class DataOut_DoFData : public DataOutInterface<patch_dim, patch_space_dim>
                   const DataPostprocessor<DoFHandlerType::space_dimension>
                     &data_postprocessor);
 
+  /**
+   * Add a multilevel data vector.
+   *
+   * This function adds the vector-valued multilevel vector @p data that belongs to
+   * the DoFHandler @p dof_handler to the graphical output. This function is
+   * typically used in conjunction with a call to set_cell_selection() that
+   * selects cells on a specific level and not the active cells (the default).
```

Comment:
Should we say here that this vector can either be the current state in a multigrid algorithm or obtained e.g. from `MGTransferMatrixFree::interpolate_to_mg()`?